recipes/trf/meta.yaml

```diff
@@ -0,0 +1,19 @@
+package:
+  name: trf
+  version: "4.07b"
+
+source:
+  fn: trf407b
+  url: https://tandem.bu.edu/trf/downloads/trf407b.linux64 # [linux]
+  url: https://tandem.bu.edu/trf/downloads/trf407b.macos64 # [osx]
```

Comment:
Very unlikely to get the source. TRF is not under an open license. 

There are open alternatives – phobos, mreps, maybe VNTRseek. Would one of those work instead?